WordPress/twentytwentyfive
# Try: Add buttons styles.

**Description**


Adding button styles according to [the designs](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=2431-1113&t=iVc8k2Kl9C1qrITw-1).

It works well in the default style. We will probably need to revisit the different variations once we have the [section styles](https://github.com/WordPress/twentytwentyfive/pull/151) in.

**Screenshots**



https://github.com/user-attachments/assets/86fb4301-931b-4d78-bed2-17b1aeb4299e

**Testing Instructions**

1. Create a page.
2. Add a button with the default style and another one with outline.
3. Confirm they're looking like [the designs](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=2431-1113&t=iVc8k2Kl9C1qrITw-1).
